app/helpers/admin/settings_helper.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Admin::SettingsHelper
+  def site_upload_delete_hint(hint, var)
+    upload = SiteUpload.find_by(var: var.to_s)
+    return hint unless upload
+
+    link = link_to t('admin.site_uploads.delete'), admin_site_upload_path(upload), data: { method: :delete }
+    safe_join([hint, content_tag(:p, link, class: 'hint')])
```

Comment:
Can it be in the same paragraph instead?

Reply:
Hm, yeah, that was kind of a leftover from the other code, resulting in weird and (I think?) wrong HTML.
Keeping the newline, though, because otherwise it feels slightly weird (and requires changing the strings to end with a dot)